src/tasks/ConnectServiceTaskTerminal.ts

```diff
@@ -120,7 +121,7 @@ export class ConnectServiceTaskTerminal extends TaskTerminalBase {
             this._writeLine(`Error: ${error.message}`);
             vscode.window.showErrorMessage(error.message);
             this.showUseKubernetesServiceEnvironmentVariablesPromptIfRequired(error.message);
-            this._logger.error(TelemetryEvent.ConnectServiceTaskTerminal_Error, error);
+            this._logger.error(TelemetryEvent.ConnectServiceTaskTerminal_Error, redactJsonObject(error));
```

Comment:
Can we add this to the other two places so this PR can address all 3 at once? Below are the other two lines to look for.
this._logger.error(TelemetryEvent.BridgeClient_ConnectError, error);
this._logger.error(TelemetryEvent.Connect_Error, error);

Reply:
💯 Sounds good, thanks, pushed now. Thank you